test/unittests/test_parameters.py

```diff
@@ -228,6 +228,10 @@ def test_diff(self):
         self.assertEqual(P1.diff(P3),
                          ({'y': 3}, {'y': 4, 'z': 4}))
 
+    def test_long_paramter(self):
+        parameters = "x = '" + ''.join(["a" for unused in range(250)]) + "'"
```

Comment:
This can be simplified a bit by using `"a" * 250` to generate a string with 250 repetitions of the letter 'a'.